modules/tinymce/src/plugins/lists/test/ts/atomic/ListNumberingTest.ts

```diff
@@ -0,0 +1,69 @@
+import { Assert, describe, it } from '@ephox/bedrock-client';
+import { Arr, Optional, OptionalInstances } from '@ephox/katamari';
+import fc from 'fast-check';
+import { parseStartValue, parseDetail, ListDetail } from 'tinymce/plugins/lists/core/ListNumbering';
+
+describe('atomic.tinymce.plugins.lists.core.ListNumberingTest', () => {
+  const tOptional = OptionalInstances.tOptional;
+
+  const check = (startValue: string, expectedDetail: Optional<ListDetail>) => {
+    const actualDetail = parseStartValue(startValue);
+
+    Assert.eq('Should convert start value to expected detail', expectedDetail, actualDetail, tOptional());
+
+    actualDetail.map(parseDetail).each((initialStartValue) => {
+      Assert.eq('Should convert detail back to initial start value', startValue, initialStartValue);
```

Comment:
Can you please use a `chai` assertion here instead? as we're trying to remove the bedrock Assert logic now so that we don't need to try and maintain a custom assertion library. Chai also offers a lot more and often clearer assertions.

Note: The one above is okay for now, as we don't have an equivalent way to compare optionals atm.